insights/tests/parsers/test_eap_json_reports.py

```diff
@@ -0,0 +1,2255 @@
+                        "wsdl-port": null,
+                        "wsdl-secure-port": null,
+                        "wsdl-uri-scheme": null,
+                        "client-config": {
+                            "Standard-Client-Config": {
+                                "post-handler-chain": null,
+                                "pre-handler-chain": null,
+                                "property": null
+                            }
+                        },
+                        "endpoint-config": {
+                            "Standard-Endpoint-Config": {
+                                "post-handler-chain": null,
+                                "pre-handler-chain": null,
+                                "property": null
+                            },
+                            "Recording-Endpoint-Config": {
+                                "post-handler-chain": null,
+                                "pre-handler-chain": {
+                                    "recording-handlers": {
+                                        "protocol-bindings": "##SOAP11_HTTP ##SOAP11_HTTP_MTOM ##SOAP12_HTTP ##SOAP12_HTTP_MTOM",
+                                        "handler": {
+                                            "RecordingHandler": {
+                                                "class": "org.jboss.ws.common.invocation.RecordingServerHandler"
+                                            }
+                                        }
+                                    }
+                                },
+                                "property": null
+                            }
+                        }
+                    },
+                    "weld": {
+                        "development-mode": false,
+                        "non-portable-mode": false,
+                        "require-bean-descriptor": false,
+                        "thread-pool-size": null
+                    }
+                }
+            }
+        },
+        "eap-deployments": {
+            "version": "1.0.0",
+            "deployments": [
+                {
+                    "name": "helloworld.war",
+                    "archives": [
+                        {
+                            "name": "helloworld.war",
+                            "version": " ",
+                            "attributes": {
+                                "sha1Checksum": "cbf0287e4457c5f6555da118fc77815c143e2d2a",
+                                "sha256Checksum": "f41848d63c4098b96beb5c3eb51139de48757738fa093aca1bf666c95608a6ca",
+                                "sha512Checksum": "70389a1e8d8581ec6706e4d731aa617d09864a602059b8bb505c68f311f83b9cc45b66f3abd25593dd573675b2df4e5fbbc1008a8f0f03fd5ae60e2f323bd85d",
+                                "deployment": "helloworld.war"
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+""".strip()
```

Comment:
I saw the check in the following test function against this Json input is quite simple, would you mind removing some un-used/un-tested keys from it to reduce the size? It seems most of them are not required.

Reply:
Removed